tests/e2e/specs/all-fields.js

```diff
@@ -23,7 +22,7 @@ const uploadMediaFile = async ( $context, fieldLabel, fileName ) => {
 	const $input = await page.$( inputSelector );
 
 	const testImagePath = path.join( __dirname, '..', 'assets', fileName );
-	const newFileName = uuid();
+	const newFileName = Math.floor( Math.random( 100000 ) * 100000 );
```

Comment:
No need to import [uuid](https://www.npmjs.com/package/uuid), we can do this in 1 line of JS.